database/gui_repo_worker.py

```
@@ -6,23 +10,37 @@
+    are properly saved.
+    """
+
+    COVER_FOLDER = ROOT_DIR / ".covers"
+
     def __init__(self):
-        self.cover_folder = ROOT_DIR / ".covers"
+        """
+        Intiates the class instance.
+        """
 
     def __enter__(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__enter__`

(ANN204)